.goreleaser.yml

```diff
@@ -41,123 +44,123 @@ dockers:
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=AGPL-3.0
     dockerfile: Dockerfile
     extra_files:
-    - config.toml.sample
-    - config-demo.toml
+      - config.toml.sample
+      - config-demo.toml
   - use: buildx
     goos: linux
     goarch: arm64
     ids:
-    - listmonk
+      - listmonk
     image_templates:
-    - "listmonk/{{ .ProjectName }}:latest-arm64v8"
-    - "listmonk/{{ .ProjectName }}:{{ .Tag }}-arm64v8"
-    - "ghcr.io/knadh/{{ .ProjectName }}:latest-arm64v8"
-    - "ghcr.io/knadh/{{ .ProjectName }}:{{ .Tag }}-arm64v8"
+      - "{{ .Env.DOCKER_ORG }}/{{ .ProjectName }}:latest-arm64v8"
+      - "{{ .Env.DOCKER_ORG }}/{{ .ProjectName }}:{{ .Tag }}-arm64v8"
+      - "ghcr.io/{{ .Env.GITHUB_ORG }}/{{ .ProjectName }}:latest-arm64v8"
+      - "ghcr.io/{{ .Env.GITHUB_ORG }}/{{ .ProjectName }}:{{ .Tag }}-arm64v8"
     build_flag_templates:
-    - --platform=linux/arm64/v8
-    - --label=org.opencontainers.image.title={{ .ProjectName }}
-    - --label=org.opencontainers.image.description={{ .ProjectName }}
-    - --label=org.opencontainers.image.url=https://github.com/knadh/{{ .ProjectName }}
-    - --label=org.opencontainers.image.source=https://github.com/knadh/{{ .ProjectName }}
-    - --label=org.opencontainers.image.version={{ .Version }}
-    - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
-    - --label=org.opencontainers.image.revision={{ .FullCommit }}
-    - --label=org.opencontainers.image.licenses=AGPL-3.0
+      - --platform=linux/arm64/v8
+      - --label=org.opencontainers.image.title={{ .ProjectName }}
+      - --label=org.opencontainers.image.description={{ .ProjectName }}
+      - --label=org.opencontainers.image.url=https://github.com/{{ .Env.GITHUB_ORG }}/{{ .ProjectName }}
+      - --label=org.opencontainers.image.source=https://github.com/{{ .Env.GITHUB_ORG }}/{{ .ProjectName }}
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=AGPL-3.0
     dockerfile: Dockerfile
     extra_files:
-    - config.toml.sample
-    - config-demo.toml
+      - config.toml.sample
+      - config-demo.toml
   - use: buildx
     goos: linux
     goarch: arm
     goarm: 6
     ids:
-    - listmonk
+      - listmonk
     image_templates:
-    - "listmonk/{{ .ProjectName }}:latest-armv6"
-    - "listmonk/{{ .ProjectName }}:{{ .Tag }}-armv6"
-    - "ghcr.io/knadh/{{ .ProjectName }}:latest-armv6"
-    - "ghcr.io/knadh/{{ .ProjectName }}:{{ .Tag }}-armv6"
+      - "{{ .Env.DOCKER_ORG }}/{{ .ProjectName }}:latest-armv6"
```

Comment:
Not necessarily. It can be self-sufficient, as provided, by setting it in line 1 but it really helps when people fork the project and need to test on their own pipeline first like I did.

See here: https://github.com/knadh/listmonk/pull/1451/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R5

Where would you want it to be documented additionally? I'll do that.

Reply:
Understood, I must've missed it while reviewing. Thanks for clarifying.

Reply:
But i need to fix the default preset now that I am looking at it. Just 1min and its fixed.

Reply:
done